itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCrudCompactorOnTez.java

```diff
@@ -826,7 +835,7 @@ public void testStatsAfterQueryCompactionOnTez() throws Exception {
     parameters = Hive.get().getTable(tblName).getParameters();
     Assert.assertEquals("The number of files is differing from the expected", "1", parameters.get("numFiles"));
     Assert.assertEquals("The number of rows is differing from the expected", "2", parameters.get("numRows"));
-    Assert.assertEquals("The total table size is differing from the expected", "735", parameters.get("totalSize"));
+    Assert.assertEquals("The total table size is differing from the expected", "736", parameters.get("totalSize"));
```

Comment:
I saw you have removed  `totalSize `   in the UT of `TestCompactor.java`, so should we remove this line as well?

Reply:
Will be handled in different ticket/PR.